### ISSUE TITLE
Invalid call mixin in Less 

### ISSUE THREAD

**Issue by mrmlnc** [jkphl/svg-sprite#133]
Hi, 

I wrote the following configuration:

```js
mode: {
  sprite: 'sprite.svg',
  css: {
    dest: 'styles/'
    common: 'icon',
    prefix: '.icon-',
    dimensions: '-wh',
    sprite: '../images/sprite.svg',
    bust: false,
    mixin: 'sprite-icon',
    render: {
      less: true
    }
  }
}
```

Gulp plugin generated for me this code:

```less
.sprite-icon() {
	background: url("../images/sprite.svg") no-repeat;
}
.icon {
	sprite-icon();  // here is missing the dot
}
// More icons
// ...
```

If I specify a dot in the property `mode.<mode>.mixin` (`mixin: '.sprite-icon'`), then:

```less
..sprite-icon() { // here two dots
	background: url("../images/sprite.svg") no-repeat;
}
.icon {
	.sprite-icon();
}
// More icons
// ...
```

**Comment by mrmlnc** [comment]
Good sounds!

**Comment by jkphl** [comment]
@mrmlnc Yep, should be fixed as of 1.2.18 ;)
